tests/core/gadget-config-defaults-to-snaps/task.yaml

```diff
@@ -156,12 +166,30 @@ execute: |
     echo "We have a model assertion"
     snap model --verbose|MATCH "model:\s* my-model-w-config"
 
-    echo "The configurable snap was installed"
+    echo "Snap with configure hook only: Installed"
     snap list|MATCH "test-snapd-with-configure${SUFFIX}"
 
-    echo "The configuration defaults from the gadget where applied"
-    snap get "test-snapd-with-configure${SUFFIX}" a|MATCH "^A$"
-    snap get "test-snapd-with-configure${SUFFIX}" b|MATCH "^B$"
+    echo "Snap with configure hook only: The config defaults from the gadget where applied by configure hook"
+    snap get "test-snapd-with-configure${SUFFIX}" a | MATCH "^A$"
+    snap get "test-snapd-with-configure${SUFFIX}" b | MATCH "^B$"
+
+    echo "Snap with default-configure and configure hooks: Installed"
+    snap list|MATCH "test-snapd-with-default-configure${SUFFIX}"
```

Comment:
You can test this by doing the following instead of using MATCH
snap list "test-snapd-with-default-configure${SUFFIX}"

Reply:
Thanks, implemented your proposal.